### DISCUSSION TITLE
What does "No installed components were detected. (0x800F1000)" mean here ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [microsoft/microsoft-ui-xaml#7775] [Q&A]
![image](https://user-images.githubusercontent.com/43340959/193426108-dc11db6f-76c7-4609-a5fd-848401d5ca73.png)

**Comment by DarranRowe** [comment]
Without context/a sample, this is an unknown.
Trying similar code in a WinUI 3 application doesn't result in an error, so it has to be related to what you are not showing.

**Reply by sigmarsson** [reply]
@DarranRowe , thanks for your message.
The code depicted was shipped by the Template Studio. What info do you need?

**Reply by sigmarsson** [reply]
@DarranRowe found. ```XamlControlsResources``` were not merged into the application resource dictionary. Perhaps a classic dictionary related error message could be better. Like "'item not found by the key specified" or something similar. Thanks again.

**Comment by sigmarsson** [comment]
This exception occurred again in a situation a control was referencing another control by ```x:Name``` but that name was not existing.

**Comment by marcelwgn** [comment]
I think this error is a wildcard error that can come from quite different things. Sometimes it gets raised by missing dictionaries, sometimes by controls referenced through x:Name not being present or visual state setters referencing missing parts.
